inspirehep/modules/orcid/api.py

```diff
@@ -68,7 +67,7 @@ def push_record_with_orcid(recid, orcid, oauth_token, put_code=None):
     orcid_api = _get_api()
 
     orcid_xml = OrcidConverter(
-        record, server_name, put_code=put_code, bibtex_citation=bibtex
+        record, get_url_for_record, put_code=put_code, bibtex_citation=bibtex
```

Comment:
We don't need to pass around the function :), you can parametrize it as an ``import_string``

Reply:
Do you mean import it in-place? Otherwise, I wanted to prevent adding more dependencies inside of the converter?

Reply:
Not in place but as an ``import_string`` in a ``config`` var, as a factory. This way it can easily change and you don't have a strong dependency.